extensions/jms/src/main/java/org/mule/extensions/jms/api/source/JmsListener.java

```diff
@@ -116,26 +112,36 @@
 
   @Parameter
   @Optional
+  @Example(EXAMPLE_CONTENT_TYPE)
   @Summary("The content type of the message body")
   private String contentType;
 
   @Parameter
   @Optional
+  @Example(EXAMPLE_ENCODING)
   @Summary("The encoding of the message body")
   private String encoding;
 
+  @Parameter
+  @Optional(defaultValue = "true")
```

Comment:
if this a good default value? This means that by default you get crappy which only makes sense if what's to gain from that is needed in most scenarios